docs/modules/develop/pages/notifications.adoc

```diff
@@ -108,6 +108,133 @@ All contents except the `email_greeting` use to require customization on each no
+
+Note that the cell returned is "decidim/notification_actions/buttons", if you want to use a custom cell, you should create it in your application and return it accordingly.
+
+The default buttons cell renders as many buttons as defined in the `action_data` array. Each button will handle a "click" event that will make an AJAX request to the URL provided in the button data. The `method` attribute is used to define the HTTP method that will be used in the AJAX request.
+
+So, it is advisable for the controller handling the request to respond with a JSON object with the following structure:
+
+[source,json]
+----
+{
+  "message": "Some message"
+}
+----
+
+Use standard HTTP status codes to indicate the result of the operation. For an example, see the implementation of the `Decidim::GroupInvitesController` that responds with a JSON object only when requests are made with the `:json` format:
+
+[source,ruby]
+----
+# decidim-core/app/controllers/decidim/group_invites_controller.rb
+
+def update
+  enforce_permission_to :accept, :user_group_invitations
+  AcceptGroupInvitation.call(inviting_user_group, current_user) do
+    on(:ok) do
+      respond_to do |format|
+        format.json do
+          render json: { message: t("group_invites.accept.success", scope: "decidim") }
+        end
+        format.all do
+          flash[:notice] = t("group_invites.accept.success", scope: "decidim")
+          redirect_to profile_groups_path(current_user.nickname)
+        end
+      end
+    end
+
+    on(:invalid) do
+      respond_to do |format|
+        format.json do
+          render json: { message: t("group_invites.accept.error", scope: "decidim") }, status: :unprocessable_entity
+        end
+        format.all do
+          flash[:alert] = t("group_invites.accept.error", scope: "decidim")
+          redirect_to profile_groups_path(current_user.nickname)
+        end
+      end
+    end
+  end
+end
+
+def destroy
+  enforce_permission_to :reject, :user_group_invitations
+
+  RejectGroupInvitation.call(inviting_user_group, current_user) do
+    on(:ok) do
+      respond_to do |format|
+        format.json do
+          render json: { message: t("group_invites.reject.success", scope: "decidim") }
+        end
+        format.all do
+          flash[:notice] = t("group_invites.reject.success", scope: "decidim")
+          redirect_to profile_groups_path(current_user.nickname)
+        end
+      end
+    end
+
+    on(:invalid) do
+      respond_to do |format|
+        format.json do
+          render json: { message: t("group_invites.reject.error", scope: "decidim") }, status: :unprocessable_entity
+        end
+        format.all do
+          flash[:alert] = t("group_invites.reject.error", scope: "decidim")
+          redirect_to profile_groups_path(current_user.nickname)
+        end
+      end
+    end
+  end
+end
+----
+
```

Comment:
```suggestion
```

Reply:
Let's remove the unecessary lines here (GH reviews don't allow me to delete them all in a single suggestion, so I'm making multiple suggestions)

Reply:
@microstudi we still have this line, can (2 empty lines between the end of the section and the next section). Can you apply this? Thanks!

Reply:
Sorry I forgot, done!